docs/proposals/algorithms/lifelong-learning/Unknown_Task_Processing_Algorithm_based_on_Lifelong_Learning_of_Ianvs.md

```diff
@@ -0,0 +1,79 @@
+# Unknown Task Processing Algorithm based on Lifelong Learning of Ianvs
```

Comment:
The name of this proposal is repeated from before, I suggest coming up with a different name.